physics/module_mp_thompson.F90

```diff
@@ -4235,7 +4269,7 @@ subroutine mp_thompson (qv1d, qc1d, qi1d, qr1d, qs1d, qg1d, ni1d,    &
            xDi = (bm_i + mu_i + 1.) * ilami
            if (xDi.lt. 5.E-6) then
             lami = cie(2)/5.E-6
-           elseif (xDi.gt. 300.E-6) then
+           elseif (xDi.gt. D0s + 100.E-6) then 
```

Comment:
Same as previously mentioned.  This line should be reverted back to the original code.

Reply:
Same as previously mentioned

Reply:
Reverted as suggested.